mindlessgen.toml

```diff
@@ -1,21 +1,21 @@
-# Default configuration for the 'Mindless Molecule GENerator' (MindlessGen) package
-# Following file locations are searched for in the following order:
+# Default configuration for the 'Mindless Molecule Generator' (MindlessGen) package
+# The following file locations are searched for in ascending order:
 # 1. Location specified by the `--config < str | Path >` command-line argument
 # 2. Current working directory (`Path.cwd()`)
 # 3. User's home directory (`Path.home()`)
 
 [general]
-# > Verbosity level defining the printout: Options: 0 = silent, 1 = default, 2 = verbose, 3 = debug
+# > Verbosity level defining the printout: Options: -1 = super-silent, 0 = silent, 1 = default, 2 = verbose, 3 = debug
 verbosity = 1
 # > Number of parallel processes to use. Corresponds to the number of physical CPU cores used. Options: <int>
 parallel = 1
-# > Maximum number of generation/optimization try-and-error cycles per molecule. Options: <int>
-max_cycles = 100
+# > Maximum number of generation & optimization try-and-error cycles per molecule. Options: <int>
+max_cycles = 200
 # > Number of molecules to generate. Options: <int>
 num_molecules = 1
-# > Do post-processing (checking for HL gap, etc.) after the optimization. Options: <bool>
+# > Do post-processing after the optimization with another engine (e.g., `orca`). Default: false. Options: <bool>
 postprocess = false
-# > Switch molecule structure XYZ writing on and off (default: true). Options: <bool>
+# > Switch molecule structure XYZ writing on and off. Default: false. Options: <bool>
 write_xyz = true
```

Comment:
`write_xyz` should than set to `false` to match the default.

Reply:
Good catch. Done.